src/components/progress-report/workflow/transitions.ts

```diff
@@ -11,78 +11,106 @@ import {
 // Therefore, these should generally flow down.
 // "back transactions" should come before/above "forward transactions".
 
+/*
+  The following Roles are always notifed for each transition and are handled within
+  progress-report-workflow-notification.handler:
+    Role.ProjectManager,
+    Role.RegionalDirector,
+    Role.FieldOperationsDirector,
+*/
 export const Transitions = defineTransitions({
   Start: {
     from: Status.NotStarted,
     to: Status.InProgress,
     label: 'Start',
     type: Type.Approve,
-    notify: 'FPM',
+    notify: {
+      TeamMembersWithRoles: [], //add additional roles desired to the array, e.g. Role.Marketing
+    },
```

Comment:
`notify` is still typed as `any`. I'd change the types and document there instead.
https://github.com/SeedCompany/cord-api-v3/blob/master/src/components/progress-report/workflow/transitions.ts#L134-L134
```ts
notify?: {
  /**
   * Notify project members with these roles
   */
  membersWithRoles?: readonly Role[];
}
```